workflow-bot-app/src/main/java/com/symphony/bdk/workflow/logs/LogsStreamingAppender.java

```diff
@@ -16,8 +20,19 @@ public class LogsStreamingAppender extends UnsynchronizedAppenderBase<ILoggingEv
 
   @Override
   protected void append(ILoggingEvent eventObject) {
+    StringBuilder formattedMessage = new StringBuilder();
+    formattedMessage.append(eventObject.getFormattedMessage());
+
+    if (eventObject.getThrowableProxy() != null) {
+      formattedMessage.append("\n")
+              .append(Arrays.stream(eventObject.getThrowableProxy().getStackTraceElementProxyArray())
+                      .map(StackTraceElementProxy::getSTEAsString)
+                      .collect(Collectors.joining("\n")));
+    }
+
+
     service.broadcast(eventObject.getTimeStamp(), eventObject.getLevel().toString(), eventObject.getLoggerName(),
-        eventObject.getFormattedMessage());
+            formattedMessage.toString());
```

Comment:
need to do a replaceAll to replace all [\r\n] with another printable character like [\t] to prevent line breaking from truncating the event source payload but allow for post-reconstruction

Reply:
Done here: https://github.com/finos/symphony-wdk/pull/255/commits/bef2534357aeb15dd44c3057534f24b6844b3881 

now the newline and carriage return are replaced by a tab character.